app/assets/javascripts/lib/ui/content/annotations/annotator.js.erb

```diff
@@ -107,7 +109,7 @@ export class Annotator extends Component {
       { annotation: attrs }, { scroll: false })
     .then( _ => this.deactivate() );
 
-    window.location = (window.location + `?annotation_id=${annotationId}`);
+    window.location.search = `annotation-id=${response.data.annotation_id}`;
```

Comment:
if the annotation-id query string isn't needed in updateAnnotation lmk and I can remove it